### PR TITLE
Generalize package name for the project (Closes #98)

### LINKED ISSUE #98: Generalize package name for the project
As for #97 we might want to generalize the package name for the project, too. So `com.snowplowanalytics.elasticsearch.loader` becomes `com.snowplowanalytics.stream.loader`.

- This naming be inline with `stream enrich` or `stream collector`
- Elasticsearch will be a module anyway and it will not be gone anywhere. we just remove a repetition.

### PR DESCRIPTION
**Pull request by szareiangm**
Just a package rename as we discussed in #98 ✋ 

CC: @Nafid

### CODE REVIEW

File: core/src/main/scala/com.snowplowanalytics.stream/loader/ElasticsearchPipeline.scala [hunk 1]
```diff
@@ -17,23 +17,19 @@
  * governing permissions and limitations there under.
  */
 
-package com.snowplowanalytics
-package elasticsearch.loader
+package com.snowplowanalytics.stream.loader
```
Comment (BenFradet) [inline core/src/main/scala/com.snowplowanalytics.stream/loader/ElasticsearchPipeline.scala]: I don't think this needs modifying, it simplifies the snowplow imports

File: core/src/main/scala/com.snowplowanalytics.stream/loader/ElasticsearchSinkApp.scala [hunk 2]
```diff
@@ -52,7 +52,7 @@ trait ElasticsearchSinkApp {
     val projectName = "snowplow-elasticsearch-loader"
     case class FileConfig(config: File = new File("."))
     val parser = new scopt.OptionParser[FileConfig](projectName) {
-      head(projectName, generated.Settings.version)
+      head(projectName, com.snowplowanalytics.stream.loader.generated.Settings.version)
```
Comment (BenFradet) [inline core/src/main/scala/com.snowplowanalytics.stream/loader/ElasticsearchSinkApp.scala]: this shouldn't need to be added

File: core/src/main/scala/com.snowplowanalytics.stream/loader/KinesisElasticsearchEmitter.scala [hunk 3]
```diff
@@ -17,15 +17,11 @@
  * governing permissions and limitations there under.
  */
 
-package com.snowplowanalytics
-package elasticsearch.loader
+package com.snowplowanalytics.stream.loader
```
Comment (BenFradet) [inline core/src/main/scala/com.snowplowanalytics.stream/loader/KinesisElasticsearchEmitter.scala]: same thing here let's leave it as is

File: core/src/main/scala/com.snowplowanalytics.stream/loader/KinesisSourceExecutor.scala [hunk 4]
```diff
@@ -17,8 +17,7 @@
   * governing permissions and limitations there under.
   */
 
-package com.snowplowanalytics
-package elasticsearch.loader
+package com.snowplowanalytics.stream.loader
```
Comment (BenFradet) [inline core/src/main/scala/com.snowplowanalytics.stream/loader/KinesisSourceExecutor.scala]: same

File: core/src/main/scala/com.snowplowanalytics.stream/loader/SnowplowTracking.scala [hunk 5]
```diff
@@ -55,7 +55,7 @@ object SnowplowTracking {
     // Not yet used
     val method = config.method
     val emitter = AsyncEmitter.createAndStart(endpoint, Some(port), config.ssl.getOrElse(false), None)
-    new Tracker(List(emitter), generated.Settings.name, appName)
+    new Tracker(List(emitter), com.snowplowanalytics.stream.loader.generated.Settings.name, appName)
```
Comment (BenFradet) [inline core/src/main/scala/com.snowplowanalytics.stream/loader/SnowplowTracking.scala]: shouldn't be needed

File: core/src/main/scala/com.snowplowanalytics.stream/loader/SnowplowTracking.scala [hunk 6]
```diff
@@ -118,7 +118,7 @@ object SnowplowTracking {
   private def trackApplicationInitialization(tracker: Tracker): Unit = {
     tracker.trackSelfDescribingEvent(SelfDescribingData(
       SchemaKey("com.snowplowanalytics.monitoring.kinesis", "app_initialized", "jsonschema", SchemaVer.Full(1,0,0)),
-      generated.Settings.name
+      com.snowplowanalytics.stream.loader.generated.Settings.name
```
Comment (BenFradet) [inline core/src/main/scala/com.snowplowanalytics.stream/loader/SnowplowTracking.scala]: same

File: core/src/main/scala/com.snowplowanalytics.stream/loader/ElasticsearchSinkApp.scala (hunk 2 above)
Comment (szareiangm) [reply core/src/main/scala/com.snowplowanalytics.stream/loader/ElasticsearchSinkApp.scala]: for some reason my code doesn't compile without it.  Maybe some sbt settings with the new laptop.

Comment (BenFradet) [reply core/src/main/scala/com.snowplowanalytics.stream/loader/ElasticsearchSinkApp.scala]: mmh weird

File: core/src/test/scala/com.snowplowanalytics.stream.loader/BadEventTransformerSpec.scala [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+      val input = """{"line":"failed","errors":["Record does not match Thrift SnowplowRawEvent schema"]}"""
+      val result = new BadEventTransformer("snowplow", "bad").fromClass(input -> JsonRecord(input, None).success)
+      val elasticsearchObject = result._2.getOrElse(throw new RuntimeException("Bad event failed transformation"))
+      elasticsearchObject.getIndex must_== "snowplow"
+      elasticsearchObject.getType must_== "bad"
+      elasticsearchObject.getSource must_== input
+    }
+  }
+
+}
```
Comment (BenFradet) [inline core/src/test/scala/com.snowplowanalytics.stream.loader/BadEventTransformerSpec.scala]: can we have the specs in different commits?

**Comment by szareiangm** [comment]
@BenFradet I addressed all of your comments. Except having specs in different commit? I think my changes for them are the same as this issue. I only changed their package name. 
But I have a small change that I want if you could please ignore it for separate commit 😃  we missed Travis config file last time when I updated Scala version. I changed it here.

**Comment by BenFradet** [comment]
ah weird I saw those specs as added code on github, merging thanks :+1:

**Comment by snowplowcla** [comment]
@szareiangm has signed the Software Grant and Corporate Contributor License Agreement

**Pull request merged**